README.md

```diff
@@ -55,7 +55,7 @@
 
 Welcome to your CLI for ELT+. It's open source, flexible, scales to your needs. Confidently move, transform, and test your data using tools you know with a data engineering workflow you’ll love.
 
-If you're a fan, star the repo ⭐️. [Plus this month, every ⭐ on GitHub removes 2 lb/~1 kg of trash from our waterways 🌊](https://meltano.com/blog/extracting-trash-to-transform-our-waterways/)
+If you're a fan, star the repo ⭐️. For every ⭐ on GitHub removes 2 lb/~1 kg of trash from our waterways 🌊.
```

Comment:
```suggestion
If you're a fan, star the repo! Every ⭐ on GitHub removes 2 lb/~1 kg of trash from our waterways 🌊.
```

Reply:
@sbalnojan I still think the "For every ⭐ on GitHub removes" sentence is awkward as is. It makes sense if it's combined with previous sentence though "If you're a fan, star the repo since every star on GitHub removes..." but a sentence starting with "For every" feels weird here.